BE/app/worker.py

```
@@ -1,15 +1,89 @@
 import logging
+import asyncio
 
 # 로깅 설정
 logger = logging.getLogger(__name__)
 
 import os
 import base64
+import uuid
+
+
+async def _save_result_to_db(topic: str, topic_request_id: str, user_id: str, channel_id: str, result: dict, formatted: dict):
```

Ruff (0.14.14)

[warning] Unused function argument: `topic`

(ARG001)